src/Compilers/Core/Portable/Emit/EmitOptions.cs

```diff
@@ -32,10 +32,9 @@ public sealed class EmitOptions : IEquatable<EmitOptions>
         public bool IncludePrivateMembers { get; private set; }
 
         /// <summary>
-        /// If not empty then compiler should enable instrumentation.
-        /// The string specifies instrumentation settings.
+        /// Type of instrumentation that should be added to the output binary.
         /// </summary>
-        public string Instrument { get; private set; }
+        public InstrumentationKind Instrument { get; private set; }
```

Comment:
I think the property should be named "InstrumentationKind". It's odd to use a verb.